From ad6fea04a18cff6bc9d34d0f46f5935a50aa3db0 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:26:14 -0400
Subject: [tools] perf: Fix test_on_exit for 32-bit build

Message-id: <1408462094-14194-206-git-send-email-jolsa@redhat.com>
Patchwork-id: 88191
O-Subject: [PATCH RHEL7.1 BZ1131394 205/325] perf tools: Fix test_on_exit for 32-bit build
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 9402802a416c96b48b2bd9331c070ba2d7550b36
Author: Adrian Hunter <adrian.hunter@intel.com>
Date: Fri Oct 18 15:29:08 2013 +0300

description
===========
builtin-record.c:42:12: error: static declaration of 'on_exit' follows non-static declaration
In file included from util/util.h:51:0,
                 from builtin.h:4,
                 from builtin-record.c:8:
/usr/include/stdlib.h:536:12: note: previous declaration of 'on_exit' was here
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/config/feature-checks/test-on-exit.c b/tools/perf/config/feature-checks/test-on-exit.c
index 8f64ed3..8e88b16 100644
--- a/tools/perf/config/feature-checks/test-on-exit.c
+++ b/tools/perf/config/feature-checks/test-on-exit.c
@@ -1,4 +1,5 @@
 #include <stdio.h>
+#include <stdlib.h>
 
 static void exit_fn(int status, void *__data)
 {
-- 
1.7.1